usaspending_api/common/matview_manager.py

```diff
@@ -9,7 +9,10 @@
 DEPENDENCY_FILEPATH = settings.APP_DIR / "database_scripts" / "matviews" / "functions_and_enums.sql"
 JSON_DIR = settings.APP_DIR / "database_scripts" / "matview_sql_generator"
 MATVIEW_GENERATOR_FILE = settings.APP_DIR / "database_scripts" / "matview_generator" / "matview_sql_generator.py"
-OVERLAY_VIEWS = [settings.APP_DIR / "database_scripts" / "matviews" / "vw_award_search.sql"]
+OVERLAY_VIEWS = [
+    settings.APP_DIR / "database_scripts" / "matviews" / "vw_award_search.sql",
+    settings.APP_DIR / "database_scripts" / "matviews" / "vw_es_award_search.sql",
```

Comment:
Is this view intended to be persistent like `vw_award_search` as opposed to only be created for the ES ETL?

Reply:
@tony-sappe, this view is only used by the ES ETL. Are you saying it can/should be created each time that stage runs and removed after it finishes?

Reply:
It could be created just for the ES awards ETL, whichever you prefer.

View will either need to be created in the nightly pipeline groovy files or the `matview_runner.groovy` file for the Jenkins job creating new matviews

Reply:
https://github.com/fedspendingtransparency/usaspending-config/pull/742